### PR TITLE
Jidea-149 show vsl 

### PR DESCRIPTION
**Pull request by absternator**
This PR displays vsl info on results page....

i have just put down generic text to display please review and let me know what to update to.

image

### CODE REVIEW

**Review by absternator — COMMENTED** [review]
Note: revert common api branch after r-api work is merged.. do NOT merge until then

File: components/CostsCard.vue
Comment (david-mears-2) [inline components/CostsCard.vue:65]: I'm no longer at all sure what the VSL value really represents. @pratikunterwegs , can you review the wording here? We might need to make it clearer that this is something variable / country-dependent, so maybe "National value of life used".

Also, I think I recall that the unit is not USD, but international dollars, but I'm not sure.

**Review by david-mears-2 — COMMENTED** [review]
I've tagged Pratik for a question.

Now that we have a table, I think a nice tidy place for the information could be in a (?) icon with tooltip, which is the way we present details about time series defintions (already in main) and how we present parameters help text in https://github.com/jameel-institute/daedalus-web-app/pull/57. I'd place this icon with tooltip into the 'Life years' row of the table.

File: components/CostsCard.vue
Comment (pratikunterwegs) [reply components/CostsCard.vue:65]: I think this is sufficient, and "Value of a statistical life" which is the expansion of VSL would also be appropriate, and perhaps preferable as it suggests that it's a broad generalisation.

It's explicitly not the national value-of-life, but a measure calculated using the country-specific GNI. It generalises well - otherwise, we'd have to implement the specific formula each country uses for its national calculations, including for countries where this isn't strictly defined or readily available.

The unit is indeed international dollars.

Comment (pratikunterwegs) [reply components/CostsCard.vue:65]: On international dollars: https://datahelpdesk.worldbank.org/knowledgebase/articles/114944-what-is-an-international-dollar

Comment (david-mears-2) [reply components/CostsCard.vue:65]:
```suggestion
      <p class="text-end fw-lighter vsl-display">
        Value of a statistical life: {{ formatCurrency(appStore.currentScenario.result.data?.average_vsl || 0) }} Int$
      </p>
```

**Comment by absternator** [comment]
> Now that we have a table, I think a nice tidy place for the information could be in a (?) icon with tooltip, which is the way we present details about time series defintions (already in main) and how we present parameters help text in #57. I'd place this icon with tooltip into the 'Life years' row of the table.

good idea however it refers to pie chart as well... so I've added the * way to show extra info 
image

File: components/CostsPie.client.vue
Comment (david-mears-2) [inline components/CostsPie.client.vue:221]: What does this affect - the label in the tooltips?

File: components/CostsTable.vue
Comment (david-mears-2) [inline components/CostsTable.vue:16]: Think this shouldn't have a space:

```suggestion
            {{ appStore.getCostLabel(childCost.id) }}<span v-if="childCost.id === 'life_years'">*</span>
```

File: components/CostsPie.client.vue
Comment (david-mears-2) [reply components/CostsPie.client.vue:226]: I don't think it looks good in the cost pie labels - sorry! I think it's best to just have the * once, in the table.

image

File: components/CostsCard.vue
Comment (EmmaLRussell) [inline components/CostsCard.vue:65]: I think this is probably one of those situations where you ought to be able to assume that if we're showing this element then the result data exists. We shouldn't ever be showing 0 here!

**Review by EmmaLRussell — APPROVED** [review]
LGTM, one tiny comment relating to our discussion on assumptions and Typescript.

**Pull request merged**
